sefaria/model/text.py

```diff
@@ -3990,8 +3990,10 @@ def padded_ref(self):
             except AttributeError:  # This is a schema node, try to get a default child
                 if self.has_default_child():
                     return self.default_child_ref().padded_ref()
+                elif self.is_book_level():
+                    raise ComplexBookLevelRefError(book_ref=self.normal())
                 else:
-                    raise InputError("Can not pad a schema node ref")
+                    raise InputError("Cannot pad a schema node ref.")
```

Comment:
I see you added a period here, but not on line 2427. Was that on purpose? I assume we don't have a clear convention on that? (yet ;-) )

Reply:
Yes, I believe it was intentional, felt like the sentence needed a period (and aligns with the other grammar fix 😄 ). Let me add on line 2427, good catch!